### PR TITLE
Bump minikube version to v1.21.0-beta.0

### PR DESCRIPTION
**Pull request by sharifelgamal**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,79 @@
 # Release Notes
 
+## Version 1.21.0-beta.0 - 2021-06-02
+Features:
+* Support setting addons from environmental variables [#11469](https://github.com/kubernetes/minikube/pull/11469)
+* Add "resume" as an alias for "unpause" [#11431](https://github.com/kubernetes/minikube/pull/11431)
+* Implement target node option for cp command [#11304](https://github.com/kubernetes/minikube/pull/11304)
```
Comment (medyagh) [inline CHANGELOG.md:7]: add back ticks for `cp`
or maybe word it like this

`cp ` command: add target node

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,79 @@
...
+
+Thank you to our triage members for this release!
+
+- afbjorklund (34 comments)
+- medyagh (32 comments)
+- andriyDev (14 comments)
+- dinever (13 comments)
+- ilya-zuyev (11 comments)
+
+Check out our [contributions leaderboard](https://minikube.sigs.k8s.io/docs/contrib/leaderboard/v1.20.0/) for this release!
```
Comment (medyagh) [inline CHANGELOG.md:74]: generate this leaderboard too
( I think it should also say `1.21-0-beta0`)

**Review by medyagh — CHANGES_REQUESTED** [review]
see comments plz

File: CHANGELOG.md (hunk 2 above)
Comment (sharifelgamal) [reply CHANGELOG.md:74]: this was automatically generated by the script, I don't have a good way to change it

Comment (sharifelgamal) [reply CHANGELOG.md:74]: actually it can be fixed manually, but we should fix the script to output this correctly

File: CHANGELOG.md (hunk 1 above)
Comment (sharifelgamal) [reply CHANGELOG.md]: done

**Review by medyagh — APPROVED** [review]
looks good, please merge after all tests look normal

**Pull request merged**